test_override.py

```diff
@@ -67,24 +67,8 @@ def test_override_InvalidUsrCPG():
         pvc = manager.create_pvc(base_yml)
         flag, base_pvc_obj = manager.check_status(30, pvc.metadata.name, kind='pvc', status='Bound',
                                                  namespace=pvc.metadata.namespace)
-        assert flag is True, "PVC %s status check timed out, not in Bound state yet..." % base_pvc_obj.metadata.name
-        logging.getLogger().info("Pvc in bound state :: %s" % base_pvc_obj.metadata.name)
-
-        #Get pvc crd details
-        pvc_crd = manager.get_pvc_crd(base_pvc_obj.spec.volume_name)
-        vol_name,vol_cpg,vol_snpCpg,vol_provType,vol_desc,vol_compr = manager.get_pvc_editable_properties(pvc_crd)
-        logging.getLogger().info(
-            "Overriden Volume properties, name::%s usrCPG::%s snpCPG::%s provType::%s compr::%s desc::%s" % (
-            vol_name,vol_cpg,vol_snpCpg,vol_provType,vol_compr,vol_desc))
-
-        # Get proprties from the array
-        hpe3par_volume = manager.get_volume_from_array(globals.hpe3par_cli, vol_name)
-        if cpg_name == vol_cpg:
-            pod = manager.create_pod(base_yml)
-            flag, pod_obj = manager.check_status(timeout, pod.metadata.name, kind='pod', status='Running',
-                                             namespace=pod.metadata.namespace)
-            assert flag is True, "Pod %s status check timed out, not in Running state yet..." % pod.metadata.name
-
+        assert flag is False, "PVC %s status check timed out, not in Bound state yet..." % base_pvc_obj.metadata.name
```

Comment:
Since the expectation here is Pvc should not come to bound, message "PVC %s status check timed out, not in Bound state yet..." should be changed. 
Something like pvc comes to bound instead of invalid override parameter 'user_cpg'.